### ISSUE TITLE
deploy deepracer-on-the-spot AMI into more regions

### ISSUE THREAD

**Issue by tylerwooten** [aws-deepracer-community/deepracer-on-the-spot#62]
deploy the AMI into 6 regions:
us-east-1
us-east-2
us-west-1
ap-south-1
eu-central-1
sa-east-1

**Comment by MarkRoss-Eviden** [comment]
@tylerwooten I'd be inclined to go eu-north1 and eu-west-2 instead of eu-central-1 and ca-central-1 instead of us-west-1.  All for cost reasons, but of course it's a moving target :-)

**Comment by MarkRoss-Eviden** [comment]
I've updated the main repo to now support lifecycle policies to delete created AMIs automatically (recently released feature).

I can't get a 'for each' loop to pass in a list of regions and distribute them, however I have managed to 'hardcode' a list.  I propose we don't merge this hardcoded list, but you use it Tyler to distribute the AMI to a bunch of regions, and then less people will have errors when using your account.  You only need to deploy in one region (which is where EC2 image builder does the build, so only costs $1/month), and then it'll be automatically distribute to each region specified, and lifecycle policies will automatically clean up old AMIs. 

Branch for you to run but not merge - https://github.com/aws-deepracer-community/deepracer-on-the-spot/tree/feature/tyler-distribution
